query_execution/PolicyEnforcerBase.hpp

```diff
@@ -179,6 +180,18 @@ class PolicyEnforcerBase {
    **/
   virtual bool admitQuery(QueryHandle *query_handle) = 0;
 
+#ifdef QUICKSTEP_DISTRIBUTED
```

Comment:
Because it also uses the code after `switch` for checking the status of a query execution.